README.md

```diff
@@ -46,6 +46,41 @@ usage:
 ```
 
 
+## Creating tables
+
+The `--output-table <file.tex>` flag converts input data into a LaTeX / PDF
+table. Conversion to PDF requires `pdflatex` to be installed. `bin/warmup_stats`
+also needs the names of the language and VM under test, and the output of
+`uname -a` on the machine the benchmarks were run on. Example usage:
+
+```
+./bin/warmup_stats  --output-table table.tex -l javascript -v V8 -u "`uname -a`" results.csv
+```
+
+
+## Creating diffs
+
+Benchmarking is often performed in order to test whether a change in a given
+VM improves or worsens its performance. Unfortunately, the difference between
+benchmark performance before and after a change is rarely simple. Users will
+want to produce a detailed comparison of the results in Krun results tables
+(above) in order to get a deeper insight into the effects of their changes.
+
+The `--output-diff <file.tex>` flag converts data from exactly two CSV files
```

Comment:
Can we clarify this first sentence? I first read it as "there's one input file called <file.tex>" (and then had to read the rest of the sentence twice to realise I was wrong).